src/lib/icon/icon.spec.ts

```diff
@@ -153,7 +153,14 @@ describe('MatIcon', () => {
       svgElement = verifyAndGetSingleSvgChild(matIconElement);
       verifyPathChildElement(svgElement, 'woof');
       expect(httpRequestUrls).toEqual(['dog.svg', 'cat.svg']);
-    });
+
+      // Assert that a registered icon can be looked-up by url.
+      iconRegistry.getSvgIconFromUrl(trust('cat.svg')).subscribe(element => {
+        expect(element.innerHTML).toContain('meow');
```

Comment:
SVGs don't have `innerHTML` on IE. This should be `verifyPathChildElement(element, 'meow');`.